compiler/mir/mirtrees.nim

```diff
@@ -148,6 +149,7 @@ type
                   ##   upper bound
     # XXX: consider using a separate operator for the slice-from-sub-sequence
     #      operation
+    mnkMutToSlice ## version of ``mnkMutToSlice`` for creating a mutable slice
```

Comment:
```suggestion
    mnkMutToSlice ## version of ``mnkToSlice`` for creating a mutable slice
```

I think it might also be better to rename this to `mnkToMutSlice`, it's perhaps more clear we're going to end up with a mutable slice, rather than drop that property.

Reply:
That seems like an improvement -- I'll change it.